### DISCUSSION TITLE
Can I just download the lyrics without replacing the local mp3 file?

### DISCUSSION THREAD

**Discussion by ueaner** [tramhao/termusic#347] [Q&A]
It's a great terminal music software, I just started to use it and I'm not very good at it yet, I downloaded a high quality mp3 file locally but without lyrics, so I searched for lyrics using `t` and downloaded it using `s`, but I found that it came down with the mp3 file and replaced my local mp3 file, so the sound quality was ruined, can I just download the lyrics without replacing the local mp3 file?

**Comment by hasezoey** [comment]
There are 2 ways currently in the tag editor but might be confusing to find them as the order of shortcut + text is reversed than the main bottom help layout:
- firstly you have to option to DOWNLOAD a full new file + lyrics via `s` (if even available, which it will not say before)
- secondly you have the option to EMBED lyrics to a *existing* file via `ENTER` (while the list is focused instead of the search options), though there are some problems i think are bad currently*1

*1 currently if you embed in the tageditor it will rename the file to something termusic thinks is correct (example `01 Alan Walker - Jump Start.mp3` -> `Alan Walker-Jump Start.mp3`) and will also clear out all other tags it might have.
